### DISCUSSION TITLE
Kernel driver build instructions outdated

### DISCUSSION THREAD

**Discussion by robertroessler** [winsiderss/systeminformer#1446] [Questions / Answers] [answered]
I missed  the nightly builds, so I started doing them myself (again) - but, assuming it is useful to give some workout to the kernel driver, it now looks like the build instructions for same are no longer quite right.
Even assuming the rest of the instructions are still relevant / correct, this step seems no longer valid:

"3. copy the bytes for public.key into the KphpTrustedPublicKey array in [verify.c](https://github.com/winsiderss/systeminformer/blob/master/KSystemInformer/verify.c)"

... is there a reasonably quick correction to the instructions that can be supplied here, or has just too much changed for this to make sense?

**Comment by jxy-s** [comment, marked as answer]
It's there, you need to replace the bytes with your key: https://github.com/winsiderss/systeminformer/blob/master/KSystemInformer/verify.c#L20

**Reply by robertroessler** [reply]
Absolutely!  Sigh, it wasn't obvious (to me, anyway) that there was a separate folder and separate SLN to find this in... unsurprisingly, this array doesn't appear in the "verify.c" in PHLib. ;)
Thanks for the non-scathing answer - and it looks like the instructions are still up-to-date.
